package.json

```diff
@@ -2,12 +2,18 @@
   "name": "ecommerce-app-predators-bn",
   "version": "1.0.0",
   "description": "predator ecommerce project",
-  "type": "module",
   "main": "app.js",
+  "type": "module",
   "directories": {
     "test": "tests"
   },
   "scripts": {
+    "make-model": "npx sequelize-cli model:generate --name user --attributes email:string",
+    "migrate": "npx sequelize-cli db:migrate",
+    "make-seed": "npx sequelize-cli seed:generate --name demo-user",
```

Comment:
can you remove "demo-user" to avoid always creating seeds with the same name, we can add a name after make-seed command